src/main/resources/templates/fragments/gps-position.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>GPS Fragment</title>
+</head>
+<body>
+    <span th:fragment="gps(point)">
+        GPS: <span th:text="${#numbers.formatDecimal(point.getY(), 0, 6)}">X</span>, <span th:text="${#numbers.formatDecimal(point.getX(), 0, 6)}">Y</span>
```

Comment:
Tady je u point.getY mezi spanem X a u point.getX Y. Má to nějaký význam, nebo je to chyba? :baby_chick:

Reply:
Obsah toho spanu je jen placeholder, který Thymeleaf nahradí. Ale zobrazí se když ten HTML soubor otevřeš přímo z disku v prohlížeči.

Ale máš pravdu, může to mást. Změním.

Díky!